src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -3002,45 +3003,86 @@ export function updateSavedUserPreferenceChartIds(
     return chartSettings;
 }
 
+export type ClinicalDataBySampleId = {
+    [sampleId: string]: { [attributeId: string]: string };
+};
+
 export async function getAllClinicalDataByStudyViewFilter(
     studyViewFilter: StudyViewFilter
-): Promise<{ [sampleId: string]: { [attributeId: string]: string } }> {
+): Promise<ClinicalDataBySampleId> {
+    console.log('getAllClinicalDataByStudyViewFilter', studyViewFilter);
```

Comment:
console

Reply:
Fixed in [7aac574](https://github.com/cBioPortal/cbioportal-frontend/commit/7aac5743da3f29903770c21cbc99cd20905a65bf)